lib/dune

```diff
@@ -3,7 +3,7 @@
  (libraries ocaml-protoc-plugin asli.libASL base64))
 
 (rule
- (targets AuxData.ml ByteInterval.ml CFG.ml CodeBlock.ml DataBlock.ml IR.ml Module.ml Offset.ml ProxyBlock.ml Section.ml Symbol.ml SymbolicExpression.ml)
+ (targets auxData.ml byteInterval.ml cFG.ml codeBlock.ml dataBlock.ml iR.ml module.ml offset.ml proxyBlock.ml section.ml symbol.ml symbolicExpression.ml)
```

Comment:
"iR" :eyes: Is there any way to configure these names? No worries if not.

Reply:
I have no idea